akka-sample-cluster-scala/src/multi-jvm/scala/sample/cluster/stats/StatsSampleSingleMasterSpec.scala

```diff
@@ -101,27 +89,30 @@ abstract class StatsSampleSingleMasterSpec extends MultiNodeSpec(StatsSampleSing
 
       Cluster(system).unsubscribe(testActor)
 
-      system.actorOf(ClusterSingletonManager.props(
-        singletonProps = Props[StatsService], terminationMessage = PoisonPill,
-        settings = ClusterSingletonManagerSettings(system).withRole("compute")),
-        name = "statsService")
-
-      system.actorOf(ClusterSingletonProxy.props(singletonManagerPath = "/user/statsService",
-        ClusterSingletonProxySettings(system).withRole("compute")),
-        name = "statsServiceProxy")
+      val singletonSettings = ClusterSingletonSettings(typedSystem).withRole("compute")
+      singletonProxy = ClusterSingleton(typedSystem).init(
+        SingletonActor(
+          Behaviors.setup[StatsService.Command] { ctx =>
```

Comment:
move this to `StatsService.apply`?

Reply:
It's here because it is different between `App` and `AppSingleMaster` and I didn't want to push that knowledge down to `StatsService` which is shared.

Reply:
ok